frontend/src/metabase/components/LoadingAndErrorWrapper.jsx

```diff
@@ -36,6 +37,24 @@ export default class LoadingAndErrorWrapper extends Component {
     messageInterval: 6000,
   };
 
+  renderError(contentClassName) {
+    if (this.props.renderError) {
+      return (
+        <div className="py4">
+          {this.props.renderError(this.getErrorMessage())}
+        </div>
```

Comment:
Avoid wrapping the error in side `contentClassName` and `.text-normal.text-light.ie-wrap-content-fix` which would make the error looks off.

#### Without the wrappers (this PR)
![image](https://user-images.githubusercontent.com/1937582/165282436-6be85672-1a3e-4088-901e-3ac0e01f3eba.png)

#### With the wrappers
![image](https://user-images.githubusercontent.com/1937582/165282587-6788f9d6-42e9-4e4b-81a1-b10ad208687c.png)